SMLHelper/Patchers/OptionsPanelPatcher.cs

```diff
@@ -51,6 +52,18 @@ internal static void AddTab_Postfix(string label, int __result)
                 modsTabIndex = __result;
         }
 
+        [PatchUtils.Prefix]
+        [HarmonyPatch(typeof(uGUI_Binding), nameof(uGUI_Binding.RefreshValue))]
+        internal static bool RefreshValue_Prefix(uGUI_Binding __instance)
```

Comment:
This patch should probably be for BZ only. Idk if this can cause any trouble in SN, but since it's needed just for BZ, I think this patch and also `ModBindingTag` should be for BZ only. Other than that, looks good to me.

Reply:
e357faf